cmd/main.go

```diff
@@ -54,7 +54,7 @@ var (
 	useBaseBt         = flag.Bool("use_base_bt", true, "Use base bigtable cache")
 	baseTableName     = flag.String("base_table", "", "Base cache Bigtable table.")
 	useImportGroup    = flag.Bool("use_import_group", false, "Use multiple base tables from import group; Only applicable when --use_base_bt=true")
-	importGroupTables = flag.String("import_group_tables", "", "A JSON string of import group tables; Only applicable when --use_import_group")
+	importGroupTables = flag.String("import_group_tables", "", "Newline separated list of import group tables; Only applicable when --use_import_group")
```

Comment:
I think most programs use "comma separated list" instead of "newline separated list", because newline might be harder to type into a command line?